jwql/website/apps/jwql/views.py

```diff
@@ -309,46 +306,18 @@ def engineering_database(request):
                 if mnemonic_identifier is not None:
                     mnemonic_query_result = query_single_mnemonic(mnemonic_identifier, start_time,
                                                                   end_time)
-                    import tempfile
-                    import sys
-                    tmpdir = tempfile.mkdtemp()
-                    mnemonic_query_result_table_html = os.path.join(tmpdir, 'mnemonic_query_result_table.html')
-                    print(mnemonic_query_result_table_html)
-                    # with open(mnemonic_query_result_table_html, 'w') as tmp:
-                    #     mnemonic_query_result.data.write(tmp, format='jsviewer')
-                    mnemonic_query_result.data[0:2].write(mnemonic_query_result_table_html, format='html')
-                    mnemonic_query_result.data[0:2].write(sys.stdout, format='html')
-                    # mnemonic_query_result.data.write(mnemonic_query_result_table_html, format='jsviewer')
                     mnemonic_query_result_plot = mnemonic_query_result.bokeh_plot()
-                    print(mnemonic_query_result_table_html)
 
             # create forms for search fields not clicked
-            form = FileSearchForm(prefix='file_search')
             mnemonic_name_search_form = MnemonicSearchForm(prefix='mnemonic_name_search')
 
-        elif 'filesearch' in request.POST.keys():
-            # Create a form instance and populate it with data from the request
-            form = FileSearchForm(request.POST, prefix='file_search')
-            if form.is_valid():
-                return form.redirect_to_files()
-
-            # create forms for search fields not clicked
-            mnemonic_name_search_form = MnemonicSearchForm(prefix='mnemonic_name_search')
-            mnemonic_query_form = MnemonicQueryForm(prefix='mnemonic_query')
-
     else:
-        form = FileSearchForm(prefix='file_search')
         mnemonic_name_search_form = MnemonicSearchForm(prefix='mnemonic_name_search')
         mnemonic_query_form = MnemonicQueryForm(prefix='mnemonic_query')
 
     template = 'engineering_database.html'
-    context = {'inst': '',
```

Comment:
This would be the reason why we're seeing that `{{ inst }}` on the EDB page now - the `inst` variable needs to be passed in the `context` dictionary as an empty string, otherwise `base.html` doesn't know what to do when it sees `{{ inst }}` in the navbar.

Reply:
Thanks. I just fixed fixed this by reverting some trimming I had done earlier.